bokehjs/test/codebase/size.ts

```diff
@@ -8,7 +8,7 @@ const LIMITS = new Map([
   // es2020
   ["js/bokeh.min.js",                1000],
   ["js/bokeh-widgets.min.js",         350],
-  ["js/bokeh-tables.min.js",          350],
+  ["js/bokeh-tables.min.js",          450],
```

Comment:
Had to increase this to pass test. I think the inclusion of the ColorMaps means it has to have the color palettes in there?

Reply:
It's possible this is due to importing from `api`, but @mattpap can also comment for certain about this.

Reply:
Changed the import and the size went back down